pkg/koordlet/metrics/psi.go

```diff
@@ -73,15 +73,15 @@ type PSIRecord struct {
 	CPUFullSupported bool
 }
 
-func getPSIRecords(psi *resourceexecutor.PSIByResource) []PSIRecord {
+func getPSIRecords(psi *system.PSIByResource) []PSIRecord {
```

Comment:
To avoid circular dependency, perhaps the metrics module should not rely on these struct types defined in other modules. Instead, try basic types like `string` and `float64`.

Reply:
util is acceptable since they are tools referenced by others